### ISSUE TITLE
Mod reply issue

### ISSUE THREAD

**Issue by alphxn** [mattmoody05/modmail.py#2]
Command raised an exception: AttributeError: 'NoneType' object has no attribute 'send'

    await user.send(embed = ReplyEmbed)
AttributeError: 'NoneType' object has no attribute 'send'

**Comment by mattmoody05** [comment]
Hi there, could you send me some information about when this is happening e.g trying to send a message to mods etc
Thanks

**Comment by alphxn** [comment]
> Hi there, could you send me some information about when this is happening e.g trying to send a message to mods etc Thanks

Yes, sure so the issue is when the Mod is trying to message back the member. To be exact in the /thread command. When we have to send the reply embed (thread command) to the user or member who has opened the modmail..

**Comment by alphxn** [comment]
Also, awesome project. I'm using this project to make my own modmail bot (not copying) just referring.

**Comment by GTHSS** [comment]
i have same error.
